README.md

```diff
@@ -1 +1,66 @@
-# LeaderBoard-
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# LeaderBoard
+
+> API TBD
+
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly add a description of the project to make your readme more professional.